src/blocks/ImageCard.tsx

```diff
@@ -197,16 +197,18 @@ const ImageCard = (props: ImageCardProps) => {
           onClose={() => setShowModal(!showModal)}
           className="image-card__overlay"
           modalClassNames="image-card__gallery-modal"
+          innerClassNames="image-card__inner-modal"
           headerClassNames="sr-only"
           closeIconColor={IconFillColors.white}
           actions={[
             <Button onClick={() => setShowModal(!showModal)} size={AppearanceSizeType.small}>
               {props.modalCloseLabel || "Close"}
             </Button>,
           ]}
+          actionsInContent
```

Comment:
I think we want the consumer to be able to customize the modal from the image card level so that Detroit doesn't have to take this style update. This would require a prop here that sets the actionsInContent prop if that makes sense?